app/App.js

```diff
@@ -10,7 +10,7 @@ import TrusteeIndex from './configs/TrusteeIndex.js';
 
 // prettier-ignore
 const routes = {
-  'marquand': 'marquand',
+  'marquand-catalogs': 'marquand',
```

Comment:
Should it have an underscore?  I though that the url we've been using is https://library.princeton.edu/marquand_catalogs/

Reply:
Since our other paths use dashes, and dashes are more idiomatic for urls, I thought it would make sense to use this opportunity to change it to dashes.